RealTimeFaceMaskDetector/LogViewer/Parser.h

```diff
@@ -0,0 +1,49 @@
+#ifndef LOGVIEWER_PARSER_H
+#define LOGVIEWER_PARSER_H
+#include <string>
+#include <vector>
+#include <fstream>
+#include <regex>
+#include "Filter.h"
+
+namespace logger {
+	class Parser {
```

Comment:
What is the difference between Parser and LogSercher? Add a few comments to them

Reply:
Already)